front/public/locales/fr/operationalStudies/manageTrainSchedule.json

```diff
@@ -50,7 +50,7 @@
   "noPlaceChosen": "Aucun point de passage/arrêt défini",
   "noOriginChosen": "Départ non défini",
   "noDestinationChosen": "Arrivée non définie",
-  "noPowerRestriction": "Aucune sélection",
```

Comment:
I think it would be better to have 2 different translations for the power restriction and the speed limit by tag :)